supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+
+### Energy Transformation
+
+Broadly, the energy transformation sectors in GCAM consist of all supplysectors between the [primary energy resources](supply_energy.html#resources) and the [final energy demands](demand_energy.html). The main energy transformation sectors highlighted in this documentation are [electricity](supply_energy.html#electricity), [refining](supply_energy.html#refining), [gas processing](supply_energy.html#en_gas_processing), [hydrogen production](supply_energy.html#hydrogen), and [district services](supply_energy.html#district-services). For more details, see [energy transformation](details_energy.html#energy-transformation)
+
+#### Electricity
+
+The GCAM electricity sector models the conversion of primary fuels (e.g., coal, gas, oil, bioenergy) to electricity. For most fuels, GCAM includes several different technology options (e.g., pulverized coal, coal IGCC, etc.). Individual technologies compete for market share based on their technological characteristics (conversion efficiency in the production of products from inputs), and cost of inputs and price of outputs. The cost of a technology in any period depends on (1) its exogenously specified non-energy cost, (2) its endogenously calculated fuel cost, and (3) any cost of emissions, as determined by the climate policy. The first term, non-energy cost, represents capital, fixed and variable O&M costs incurred over the lifetime of the equipment (except for fuel or electricity costs). For electricity technologies, GCAM reads in each of these terms and computes the levelized cost of energy within the model. For example, the non-energy cost of coal-fired power plant is calculated as the sum of overnight capital cost (amortized using a capital recovery factor and converted to dollars per unit of energy output by applying a capacity factor), fixed and variable operations and maintenance costs. The second term, fuel or electricity cost, depends on the specified efficiency of the technology, which determines the amount of fuel or electricity required to produce each unit of output, as well as the cost of the fuel or electricity. For more information, see [electricity](details_energy.html#electricity).
+
+#### Refining 
+
+The refining sector, or liquid fuels production sector, explicitly tracks all energy inputs, emissions, and costs involved with converting primary energy forms into liquid fuels. Liquid fuels include gasoline, diesel, kerosene, ethanol and many other liquid hydrocarbon fuels; for the full mapping see [Mapping the IEA Energy Balances](energy.html#mapping-the-iea-energy-balances). The refining sector includes subsectors of oil refining, biomass liquids, gas to liquids, and coal to liquids, each of which are described below. Each of these four subsectors is available starting in the first future time period, and the capital stocks of refineries are explicitly tracked.  For more information, see [refining](details_energy.html#refining).
```

Comment:
I wonder whether we should be explicit that although liquid fuels encompasses many products, GCAM only models a single "refined liquids" product that is consumed by all end-use sectors.